### ISSUE TITLE
Precompile OrdinaryDiffEq failed

### ISSUE THREAD

**Issue by Adamszk** [SciML/DifferentialEquations.jl#608]
Hi,
I am using OpenSuse Leap 15.   Julia 1.0.5. (later version of julia does not work) I installed Differential equations.jl and it failed to precompile.  So I went for OrdinaryDiffEq package and it failed to precompile.  How do I fix it?

julia> using OrdinaryDiffEq
[ Info: Precompiling OrdinaryDiffEq [1dea7af3-3e70-54e6-95c3-0bf5283fa5ed]
WARNING: Method definition vec(Number) in module FiniteDiff at /home/adam/.julia/packages/FiniteDiff/EBys0/src/jacobians.jl:128 overwritten in module DiffEqDiffTools at /home/adam/.julia/packages/DiffEqDiffTools/uD0fb/src/jacobians.jl:114.
ERROR: LoadError: LoadError: syntax: incomplete: premature end of input
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1044
 [3] include at ./sysimg.jl:29 [inlined]
 [4] include(::String) at /home/adam/.julia/packages/OrdinaryDiffEq/8Pn99/src/OrdinaryDiffEq.jl:1
 [5] top-level scope at none:0
 [6] include at ./boot.jl:317 [inlined]
 [7] include_relative(::Module, ::String) at ./loading.jl:1044
 [8] include(::Module, ::String) at ./sysimg.jl:29
 [9] top-level scope at none:2
 [10] eval at ./boot.jl:319 [inlined]
 [11] eval(::Expr) at ./client.jl:393
 [12] top-level scope at ./none:3
in expression starting at /home/adam/.julia/packages/OrdinaryDiffEq/8Pn99/src/tableaus/rkc_tableaus.jl:12393
in expression starting at /home/adam/.julia/packages/OrdinaryDiffEq/8Pn99/src/OrdinaryDiffEq.jl:106
ERROR: Failed to precompile OrdinaryDiffEq [1dea7af3-3e70-54e6-95c3-0bf5283fa5ed] to /home/adam/.julia/compiled/v1.0/OrdinaryDiffEq/DlSvy.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1203
 [3] _require(::Base.PkgId) at ./loading.jl:960
 [4] require(::Base.PkgId) at ./loading.jl:858
 [5] require(::Module, ::Symbol) at ./loading.jl:853

julia> Pkg.status()
    Status `~/.julia/environments/v1.0/Project.toml`
  [0c46a032] DifferentialEquations v6.10.1
  [7073ff75] IJulia v1.21.2
  [1dea7af3] OrdinaryDiffEq v5.29.0

**Comment by ChrisRackauckas** [comment]
We highly recommend using the current Julia release, not v1.0.5. However, the fact that this isn't working is odd. Could you try removing and adding again? "premature end of input" seems like it didn't download the whole file.

**Comment by Adamszk** [comment]
I downloaded 1.4.1 version again and try to install it.  When I run julia it failed,
Invalid instruction at 0x7faaffccd9f8: 0x66, 0x0f, 0x3a, 0x0f, 0xc0, 0x08, 0x0f, 0x83, 0x61, 0x05, 0x00, 0x00, 0x48, 0xc1, 0xe0

signal (4): Illegal instruction
in expression starting at none:0
_ZN4llvm13LexicalScopes23assignInstructionRangesERNS_15SmallVectorImplISt4pairIPKNS_12MachineInstrES5_EEERNS_8DenseMapIS5_PNS_12LexicalScopeENS_12DenseMapInfoIS5_EENS_6detail12DenseMapPairIS5_SB_EEEE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
_ZN4llvm13LexicalScopes10initializeERKNS_15MachineFunctionE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
_ZN4llvm18LiveDebugVariables20runOnMachineFunctionERNS_15MachineFunctionE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
_ZN4llvm19MachineFunctionPass13runOnFunctionERNS_8FunctionE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
_ZN4llvm13FPPassManager13runOnFunctionERNS_8FunctionE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
_ZN4llvm13FPPassManager11runOnModuleERNS_6ModuleE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
_ZN4llvm6legacy15PassManagerImpl3runERNS_6ModuleE at /home/adam/julia-1.4.1/bin/../lib/julia/libLLVM-8jl.so (unknown line)
operator() at /buildworker/worker/package_linux64/build/src/jitlayers.cpp:356
addModule at /buildworker/worker/package_linux64/build/usr/include/llvm/ExecutionEngine/Orc/IRCompileLayer.h:84 [inlined]
addModule at /buildworker/worker/package_linux64/build/src/jitlayers.cpp:485
jl_add_to_ee at /buildworker/worker/package_linux64/build/src/jitlayers.cpp:747 [inlined]
jl_finalize_function at /buildworker/worker/package_linux64/build/src/jitlayers.cpp:755
getAddressForFunction at /buildworker/worker/package_linux64/build/src/codegen.cpp:1414
jl_generate_fptr at /buildworker/worker/package_linux64/build/src/codegen.cpp:1508
jl_compile_method_internal at /buildworker/worker/package_linux64/build/src/gf.c:1912
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2153 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2322
#764 at ./client.jl:370
jfptr_#764_8156 at /home/adam/julia-1.4.1/lib/julia/sys.so (unknown line)
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2144 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2322
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1700 [inlined]
do_apply at /buildworker/worker/package_linux64/build/src/builtins.c:643
jl_f__apply_latest at /buildworker/worker/package_linux64/build/src/builtins.c:693
#invokelatest#1 at ./essentials.jl:712 [inlined]
invokelatest at ./essentials.jl:711 [inlined]
run_main_repl at ./client.jl:367
exec_options at ./client.jl:305
_start at ./client.jl:484
jfptr__start_2075 at /home/adam/julia-1.4.1/lib/julia/sys.so (unknown line)
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2144 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2322
jl_apply at /buildworker/worker/package_linux64/build/ui/../src/julia.h:1700 [inlined]
true_main at /buildworker/worker/package_linux64/build/ui/repl.c:96
main at /buildworker/worker/package_linux64/build/ui/repl.c:217
__libc_start_main at /lib64/libc.so.6 (unknown line)
_start at julia (unknown line)
Allocations: 2505 (Pool: 2495; Big: 10); GC: 0

**Comment by ChrisRackauckas** [comment]
Is this the generic binary from https://julialang.org/downloads/ ?

**Comment by Adamszk** [comment]
Yes

**Comment by ChrisRackauckas** [comment]
If you do 

```julia
]add LoopVectorization
]test LoopVectorization
```

is it good?

**Comment by Adamszk** [comment]
I am sorry, I do not know where do I type ]add ...etc  .

**Comment by Adamszk** [comment]
I assume that I have to work with working version of julia or version 1.0.5 and then use pkg.add....?

**Comment by Adamszk** [comment]
I did
 using Pkg
Pkg.add ("LoopVectorization")
It went okay
next thing I did,
 using Pkg
Pkg.test ("LoopVectorization")
after a while it failed   it is lengthy display  but it starts with vfmadd_fast not found.

**Comment by ChrisRackauckas** [comment]
Interesting, so that doesn't work but fails in a different way...

Could you try:

```julia
using Pkg; Pkg.add("BugReporting")
using BugReporting
BugReporting.make_interactive_report("rr")
using DifferentialEquations
```

?

**Comment by giordano** [comment]
I think it's https://github.com/JuliaLang/julia/issues/35215.  What CPU do you have?

**Comment by Adamszk** [comment]
I am using
OS:      openSuse Leap 15.0
CPU:    2 x Intel Pentium D CPU 3.4 GHz
RAM:   2.9 GB Ram

adding  package bugreporting failed. see below

julia> using Pkg

julia> Pkg.add("BugReporting")
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Tar [a4e569a6]:
 Tar [a4e569a6] log:
 ├─possible versions are: [0.1.0, 0.2.0-0.2.1, 1.0.0-1.0.1, 1.1.0, 1.2.0, 1.3.0-1.3.1, 1.4.0] or uninstalled
 ├─restricted by compatibility requirements with BugReporting [bcf9a6e7] to versions: [1.3.1, 1.4.0]
 │ └─BugReporting [bcf9a6e7] log:
 │   ├─possible versions are: 0.1.0-0.1.1 or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions 0.1.0-0.1.1
 └─restricted by julia compatibility requirements to versions: [0.1.0, 0.2.0-0.2.1] or uninstalled — no versions left
Stacktrace:
 [1] #propagate_constraints!#61(::Bool, ::Function, ::Pkg.GraphType.Graph, ::Set{Int64}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:1007
 [2] propagate_constraints! at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:948 [inlined]
 [3] #simplify_graph!#121(::Bool, ::Function, ::Pkg.GraphType.Graph, ::Set{Int64}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:1462
 [4] simplify_graph! at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:1462 [inlined] (repeats 2 times)
 [5] resolve_versions!(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}, ::Nothing) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:373
 [6] resolve_versions! at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:316 [inlined]
 [7] #add_or_develop#62(::Array{Base.UUID,1}, ::Symbol, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1201
 [8] #add_or_develop at ./none:0 [inlined]
 [9] #add_or_develop#15(::Symbol, ::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:69
 [10] #add_or_develop at ./none:0 [inlined]
 [11] #add_or_develop#14 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:34 [inlined]
 [12] #add_or_develop at ./none:0 [inlined]
 [13] #add_or_develop#11 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:33 [inlined]
 [14] #add_or_develop at ./none:0 [inlined]
 [15] #add_or_develop#10 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:32 [inlined]
 [16] #add_or_develop at ./none:0 [inlined]
 [17] #add#20 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:74 [inlined]
 [18] add(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:74
 [19] top-level scope at none:0

**Comment by ChrisRackauckas** [comment]
Is the problem the Pentium D?

**Comment by giordano** [comment]
I think so.  The invalid instruction is an SSSE3 one, but as far as I can tell from https://en.wikipedia.org/wiki/Pentium_D this set is not supported.  Here you can see the CPUs supporting SSSE3: https://en.wikipedia.org/wiki/SSSE3#CPUs_with_SSSE3

**Comment by giordano** [comment]
In https://github.com/JuliaLang/julia/issues/35215#issuecomment-602898940 it was noted that compiling Julia from source should work though, see https://github.com/JuliaLang/julia/blob/f705e6026cb3913de7dbb3f7992bd96031428d84/doc/build/build.md.  You probably need to set
```
USE_BINARYBUILDER=0
```

**Comment by Adamszk** [comment]
I am obsolete 
I think my cpu support ssse2 only.

**Comment by giordano** [comment]
That's fine, unfortunately the official binary comes with a build of LLVM that requires a newer instruction set, but if you're willing to build Julia from source you should be able to use it anyway.

**Comment by Adamszk** [comment]
Now I am trying to figure out how to build julia from scratch.  I see the instructions there
https://github.com/JuliaLang/julia 
but I do not see instructions how to add your suggestion use_binarybuilder=0

**Comment by giordano** [comment]
It's in the page I linked above: https://github.com/JuliaLang/julia/blob/f705e6026cb3913de7dbb3f7992bd96031428d84/doc/build/build.md#building-julia

**Comment by Adamszk** [comment]
I am not quite clear on that link...
after running 
git checkout v1.4.1
which will create a make file
I open it with editor and add the suggested line use_binarybuilder=0 am I right?

**Comment by Adamszk** [comment]
Or should I type this: 
 git checkout v1.4.1
echo"USEICC = 0" >> Make.user
echo"USEIFC = 0" >> Make.user
echo"USE_INTEL_MKL = 1" >> Make.user
echo"USE_INTEL_LIBM = 0" >> Make.user
​echo"use_binarybuilder=0">>Make.user
source /opt/intel/bin/compilervars.sh intel64
make-j$((`nproc`-1))1>build_log.txt 2>build_error.txt
Is this correct?

**Comment by giordano** [comment]
You have to add the line
```
USE_BINARYBUILDER=0
```
to the file `Make.user` in the top-level directory of the Julia source code.  Create this file if it doesn't exist

**Comment by Adamszk** [comment]
Ok. I will let you know tomorrow how build will play out. Thanks.

**Comment by Adamszk** [comment]
I have error message on first command,

adam@linux-kyb0:~/julia> git checkout v1.4.1
error: Your local changes to the following files would be overwritten by checkout:
        Make.inc
Please commit your changes or stash them before you switch branches.
Aborting
adam@linux-kyb0:~/julia> 

Note: I've made changes to make.inc by adding a following line
USE_BINARYBUILDER:=0
and setting flags from zero to one in the following lines
USE_INTEL_MKL ?= 1
USE_INTEL_LIBM  ?= 1
I have intel cpu and I assume (if I am correct) that I have those libraries with extension of .so.  I used find command and found .so libraries usually scattered around.

why check out did not go through?

**Comment by Adamszk** [comment]
I proceeded with make and it did not go through

/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cannot find -lmkl_rt

**Comment by giordano** [comment]
Maybe let's go simple and don't use mkl?

**Comment by Adamszk** [comment]
looks like your suggestion is correct.   I flagged back from 1 to 0 and make again the same directory.
the make is proceeding right now and giving me a lot of warnings related to int and max.   Process still going through at this moment.

**Comment by Adamszk** [comment]
It did not compile successfully but I made some progress so far.
I was compiling probably the latest version instead version 1.4.1
I am displaying only portion of my compilation results:

 OpenBLAS build complete. (BLAS CBLAS LAPACK LAPACKE)

  OS               ... Linux             
  Architecture     ... x86_64               
  BINARY           ... 64bit                 
  Use 64 bits int    (equivalent to "-i8" in Fortran)      
  C compiler       ... GCC  (command line : gcc -m64)
  Fortran compiler ... GFORTRAN  (command line : gfortran -m64)
  Library Name     ... libopenblas64_p-r0.3.9.a (Multi threaded; Max num-threads is 16)

To install the library, you can run "make PREFIX=/path/to/your/installation install".

While it was compiling it, I was getting annoying message:
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
        LANGUAGE = "en_US:pl",
        LC_ALL = (unset),
        LC_CTYPE = "en_US.UTF-8,LANG=en_US.UTF-8",
        LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_US.UTF-8").
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
        LANGUAGE = "en_US:pl",
        LC_ALL = (unset),
        LC_CTYPE = "en_US.UTF-8,LANG=en_US.UTF-8",
        LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_US.UTF-8").
OK.

After successful blas compilation, I got type 1 and 2 errors:

cholmod_super_solve.c:(.text+0x12a9): undefined reference to `dtrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x1380): undefined reference to `dgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x14e9): undefined reference to `dgemv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x153c): undefined reference to `dtrsv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x170f): undefined reference to `ztrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x187f): undefined reference to `zgemv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x18e0): undefined reference to `ztrsv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x19b1): undefined reference to `zgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_l_super_numeric.o: in function `cholmod_l_super_numeric':
cholmod_super_numeric.c:(.text+0x17a7): undefined reference to `zherk_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x18c3): undefined reference to `zgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x1ac7): undefined reference to `zpotrf_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x1cef): undefined reference to `ztrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x21bf): undefined reference to `dsyrk_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x22dd): undefined reference to `dgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x24c3): undefined reference to `dpotrf_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x2a97): undefined reference to `zherk_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x2bb3): undefined reference to `zgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x2dbb): undefined reference to `zpotrf_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x2fcf): undefined reference to `ztrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_numeric.c:(.text+0x319c): undefined reference to `dtrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_l_super_solve.o: in function `cholmod_l_super_lsolve':
cholmod_super_solve.c:(.text+0x4ef): undefined reference to `dtrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x5cf): undefined reference to `dgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x886): undefined reference to `ztrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x91e): undefined reference to `dtrsv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x9a9): undefined reference to `dgemv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0xb4d): undefined reference to `zgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0xc10): undefined reference to `ztrsv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0xca5): undefined reference to `zgemv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_l_super_solve.o: in function `cholmod_l_super_ltsolve':
cholmod_super_solve.c:(.text+0x1207): undefined reference to `dtrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x12e7): undefined reference to `dgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x1565): undefined reference to `ztrsm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x16fb): undefined reference to `zgemm_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x17bc): undefined reference to `zgemv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x181d): undefined reference to `ztrsv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x18cd): undefined reference to `dgemv_'
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cholmod_super_solve.c:(.text+0x191e): undefined reference to `dtrsv_'
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:544: /home/adam/julia/deps/scratch/SuiteSparse-5.4.0/lib/libcholmod.so.3.0.13] Error 1
make[3]: *** [Makefile:31: library] Error 2
make[2]: *** [Makefile:19: library] Error 2
make[1]: *** [/home/adam/julia/deps/suitesparse.mk:62: scratch/SuiteSparse-5.4.0/build-compiled] Error 1
make: *** [Makefile:60: julia-deps] Error 2

**Comment by giordano** [comment]
I'd recommend to start from scratch from a completely clean state, my feeling is that having started with MKL you may have some configurations wrong.

If this fails, I'd suggest you to continue this discussion on Discourse: https://discourse.julialang.org/  This issue doesn't really belong here anyway.

**Comment by Adamszk** [comment]
Understood.  I will do again with a clean state and will report.   Question: Which version 1.4.1 , 1.4.2 or later?

**Comment by giordano** [comment]
Either 1.4.1 or 1.4.2 should be fine

**Comment by chriselrod** [comment]
I have a local version that was able to pass tests on Julia 1.0 a few weeks ago, but I never merged it into master.
But I could make a release

> Thanks for followup and your time....
> I did
> using Pkg
> Pkg.add ("LoopVectorization")
> It went okay
> next thing I did,
> using Pkg
> Pkg.test ("LoopVectorization")
> after a while it failed it is lengthy display but it starts with vfmadd_fast not found.

Which version of Julia?
Which version of LoopVectorization?
Which version of SIMDPirates?

Only very out-dated versions of LoopVectorization allowed Julia 1.0 within it's `Project.toml`, but it never passed tests on Julia 1.0. `vfmadd_fast` not found sounds like it's using a very old SIMDPirates.
I do have a local version of LoopVectorization that did pass tests on Julia 1.0, but the tests take too long on Julia 1.0 for Travis (Julia 1.4 compiles far faster) so I never bothered merging it into master. But I could release it if people really want Julia 1.0.

**Comment by Adamszk** [comment]
Never mind old version 1.0.5.
My report as promised.
As suggested by giordano, I build 1.4.2.  It took me 6 hours. Success.  Then I build OrdinaryDiffEq pkg first.  It took me about 10 minutes.  Success. Then I went to build DifferentialEquation pkg and it was building for 3 hours.  I stopped the process there.  Not Done.  It was too much for me.  I installed differential equation tutorials and kernel for jupyter.  Success.  When I run introductory example, the simple line like print hello takes too long.  I guess my system and cpu is not for julia.  At least i managed to have kernel running and you know that CPU: 2 x Intel Pentium D CPU 3.4 GHz RAM: 2.9 GB Ram is just too old.  Thanks for help and your time.
